pkg/insights/types/types.go

```diff
@@ -56,4 +58,5 @@ type InsightsRecommendation struct {
 	ErrorKey    string
 	Description string
 	TotalRisk   int
+	ClusterID   v1.ClusterID
```

Comment:
This is not a bad idea at all. I am just thinking if for example setting the cluster version once in the `RecommendationCollector` would be an option to consider.

Reply:
I will look into it.